### ISSUE TITLE
evaluation

### ISSUE THREAD

**Issue by LucasSpider** [deepset-ai/haystack-tutorials#168]
In the tutorial 09_DPR_training.ipynb, how to add evaluation indicators when fine-tuning DPR, is there any code provided? Like the acc\f1\loss in the tutorial09

**Comment by sandeepgsc** [comment]
I have the same question. Please somebody help regarding validation loss and accuracy.

**Comment by bilgeyucel** [comment]
Closing this issue as we're not supporting Haystack v1 anymore 
Haystack 2.0 release announcement: https://haystack.deepset.ai/blog/haystack-2-release
Migration guide: https://docs.haystack.deepset.ai/docs/migration
